premake5.lua

```diff
@@ -68,6 +68,11 @@ filter({"configurations:Debug", "platforms:Windows"})
     "/NODEFAULTLIB:MSVCRTD",
   })
 
+filter({"configurations:Debug", "platforms:Linux"})
+  buildoptions({
+    "-g",
```

Comment:
Since this is a part of another pull request - can you create a separate pull request with _just_ this change so we can merge it? Appreciated!

Reply:
Done in #1341